flink-table/flink-table-common/src/main/java/org/apache/flink/table/expressions/CallExpression.java

```diff
@@ -104,6 +191,11 @@ public String getFunctionName() {
         }
     }
 
+    public CallExpression replaceArgs(List<ResolvedExpression> args, DataType dataType) {
+        return new CallExpression(
+                isTemporary, functionIdentifier, functionDefinition, args, dataType);
```

Comment:
nit: I would use `this.` to be 100% clear which come from this and which are replaced.